pkg/gc/config.go

```diff
@@ -0,0 +1,79 @@
+/*
+Copyright 2018 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package gc
```

Comment:
tests?

In particular, can you add a similar test to [this](https://github.com/knative/serving/blob/c78f67e67db0337fc686c9491b53b171d482f31b/pkg/logging/config_test.go#L161-L177) to test the checked in config file?

Reply:
done